File: week6/cpp/medium.cpp
#include <bits/stdc++.h> // Include all standard C++ libraries
using namespace std; // Use the standard namespace
 
void solve(){
		int a[3];
		for(int i=0; i<3; i++)	cin>>a[i];
 
		sort(a, a+3);
 
		if(a[1] == a[0] || a[1] == a[2])	cout<<-1<<"\n";
		else cout<<a[1]<<"\n";
		
}
 
int main(){
	
	int tstCnt; // Declare an integer variable for the number of test cases
 
	cin>>tstCnt; // Read the number of test cases from the input
 
	while(tstCnt--){ // Loop for each test case
 
		solve(); // Call the solve function
 
	}
 
	return 0;
}
 

File: week6/py/medium.py
def solve():
    a = list(map(int, input().split()))
    a.sort()

    if a[1] == a[0] or a[1] == a[2]:
        print(-1)
    else:
        print(a[1])


tstCnt = int(input())

for _ in range(tstCnt):
    solve()

